test/test_torch.py

```diff
@@ -5588,10 +5587,10 @@ def test_index_add(self):
                             dest = make_tensor(dest.shape, device=device, dtype=dest.dtype, noncontiguous=True)
                         src = torch.randn(num_copy, *other_sizes, device=device)
                         if not src_contig:
-                            src = torch.testing.make_non_contiguous(src)
+                            src = noncontiguous_like(src)
```

Comment:
Not using `torch.testing.make_tensor` here since `src` samples from a normal distribution and the former would sample from a uniform one. If the intent is to just sample positive as well as negative values, LMK.